Move.toml

```diff
@@ -7,7 +7,7 @@ authors = ["Native"]
 [dependencies]
 # For remote import, use the `{ git = "...", subdir = "...", rev = "..." }`.
 # Revision can be a branch, a tag, and a commit hash.
-btc_parser = { git = "https://github.com/gonative-cc/sui-native", subdir = "./btc_parser", rev = "master" }
+btc_parser = { git = "https://github.com/gonative-cc/sui-native", subdir = "./btc_parser", rev = "vu/version-manager" }
```

Comment:
Why not master?

Reply:
The version in the master branch has a problem when I deploy to devnet, testnet...  
The problem is a non-zero address for btc_parser and light client can not publish.